about/exploring.md

```diff
@@ -18,16 +20,19 @@ The fact that over the past few years, _billions_ of new devices have come onlin
 ## Why embedded browsers are different
 For many of us, a browser is an application like the one you're probably using now.  You click an icon on your graphical operating system, navigate somewhere with a URL bar, search, and so on.  You have bookmarks and tabs that you can drag around, and lots of other features.
```

Comment:
I think we could introduce here the “OS“ acronym, which gets used below:

> You click an icon on your graphical operating system <strong>(OS)</strong>, navigate...

Reply:
done